NEWS.md

```diff
@@ -0,0 +1,56 @@
+
+# MMAction2 Release Notes
+
+## Master
+
+### Models
+* Add NonLocal module in TSM and I3D ([#41](https://github.com/open-mmlab/mmaction2/pull/41))
+* Support SSN
+    - Add pipeline ([#33](https://github.com/open-mmlab/mmaction2/pull/33))
+    - Add losses ([#52](https://github.com/open-mmlab/mmaction2/pull/52))
```

Comment:
In the final release note, it would be better like `Add SSN. (#33, #52)` since users care more about SSN but not SSN losses, SSN pipeline.

Reply:
yeah. we may need to find a good balance to please both contributors and end users. listing prs (and corresponding contributors) in news is a way of acknowledgement

Reply:
(mainly for new contributors

Reply:
Agree. For a single model, howerver, multiple PRs are usually created by the same contributor.